yaml/com/ibm/PLDM/ChapData.interface.yaml

```diff
@@ -0,0 +1,18 @@
+description: >
+    Implement this interface to pass Chap data details to Host which are set by
+    the user.
+
+    com.ibm.PLDM.ChapData object implements this interface.
+
+properties:
+    - name: ChapName
+      type: string
+      description: >
+          ChapName is a user selected name associated with each of the chap
+          secret password.
```

Comment:
Pls change "password" to "user challenge" as we wanted to avoid the word password

Reply:
done